frontend/src/views/dashboard/widgets/WordsLearned.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import Divider from '../../../components/Divider';
+
+const WordsLearned = () => {
+  const wordList = [
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    '8',
+    '9',
+    '10',
+    '11',
+    '12',
+    '13',
+    '14',
+    '15',
+    '16',
+    '17',
+    '18',
+    '19',
+    '20',
+  ]
+  return (

```

Comment:
```suggestion
  ];
  
  return (

```
new before return and semi colon